djangocms_pageadmin/admin.py

```diff
@@ -427,9 +435,89 @@ def get_urls(self):
                 self.admin_site.admin_view(self.set_home_view),
                 name="{}_{}_set_home_content".format(*info),
             ),
+            re_path(
+                r'^export_csv/$',
+                self.admin_site.admin_view(self.export_to_csv),
+                name="{}_{}_export_csv".format(*info),
+            ),
         ]
         return new_urls + old_urls
 
+    def _format_export_datetime(self, date):
+        """
+        date: DateTime object
+        date_format: String, date time string format for strftime
+
+        Returns a formatted human readable date time string
+        """
+        if isinstance(date, datetime.date):
+            return date.strftime("%Y/%m/%d %H:%M %z")
+        return ""
+
+    def export_to_csv(self, request):
+        """
+        Retrieves the queryset and exports to csv format
+        """
+        queryset = self.get_exported_queryset(request)
+        meta = self.model._meta
+        field_names = ['Title', 'Expiry Date', 'Version State', 'Version Author', 'Url',
+                       'Compliance Number']
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename={}.csv'.format(meta)
+        writer = csv.writer(response)
+        writer.writerow(field_names)
+        for row in queryset:
+            title = row.title
+            expiry_date = self._format_export_datetime(self.expiry_date(row))
+            version_state = self.state(row)
+            version_author = self.author(row)
+            url = self.url(row, True)
+            compliance_number = self.compliance_number(row)
+            writer.writerow([title, expiry_date, version_state, version_author, url, compliance_number])
+
+        return response
+
+    def expiry_date(self, obj):
```

Comment:
```suggestion
    def get_expiry_date(self, obj):
```
For consistency with other similar ModelAdmin methods like these, I would consider prefixing this and the `compliance_number` method below with `get`